solr/solr-ref-guide/modules/deployment-guide/pages/solr-control-script-reference.adoc

```diff
@@ -1634,16 +1634,27 @@ NOTE: The `export` command only works with in a Solr running in cloud mode.
 
 The `bin/solr export` command takes the following parameters:
 
-`--url <url>`::
+`--solr-url <url>`::
 +
 [%autowidth,frame=none]
 |===
 s|Required |Default: none
```

Comment:
Is the default `none` or the local host for `--solr-url`?

Reply:
nice